Source/Public/String+Emoji.swift

```diff
@@ -20,16 +20,20 @@ extension CharacterSet {
     static let asciiPrintableSet = CharacterSet(charactersIn: "!\"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~")
 }
 
-extension Unicode.Scalar {
+extension UInt32 {
     static let cancelTag: UInt32 = 0xE007F
+}
+
+extension Unicode.Scalar {
+    static let cancelTag: Unicode.Scalar = Unicode.Scalar(UInt32.cancelTag)!
```

Comment:
A nitpick but we could get rid of extension on `UInt32` by defining the constant explicitly:

```suggestion
    static let cancelTag: Unicode.Scalar = Unicode.Scalar(0xE007F)!
```

and use `Unicode.Scalar.cancelTag.value` in the switch.